app/upgrades/v21/upgrades.go

```diff
@@ -159,6 +159,7 @@ func CreateUpgradeHandler(
 		// Set expedited proposal param:
 		govParams := keepers.GovKeeper.GetParams(ctx)
 		govParams.ExpeditedMinDeposit = sdk.NewCoins(sdk.NewCoin("uosmo", sdk.NewInt(5000000000)))
+		govParams.MinInitialDepositRatio = "0.250000000000000000"
```

Comment:
Just curious, why do we need to set this again? Is this a new param introduced in v0.47?

Reply:
Yeah, when we introduced this we hard coded it I believe. Since we are moving to the SDKs implementation, they use a param, so we need to actually set this param (defaults to zero otherwise)